src/haddock/modules/topology/topocg/defaults.yaml

```diff
@@ -1 +1,182 @@
-{}
+cgffversion:
+  default: 2
+  type: integer
+  min: 2
+  max: 3
+  choices:
+    - 2
+    - 3
+  title: MARTINI Force-field version
+  short: Force-field version for the coarse-grain mapping and parameters
+  long: CNS, the computational engine used by HADDOCK can generate a lot of output
+    messages. This parameter controls the verbosity of CNS (verbose, normal or quiet).
+  group: module
+  explevel: expert
+autohis:
+  default: true
+  type: boolean
+  title: Automatic HIS protonation state
+  short: 'The protonation state of histidine (+1: HIS or 0: HISD/HISE) will be automatically
+    set by HADDOCK'
+  long: 'If set to true, HADDOCK will automatically define the protonation state of
+    histidines ((+1: HIS or 0: HISD/HISE) by selecting the state leading to the lowest
+    electrostatic energy'
+  group: molecule
+  explevel: expert
+delenph:
+  default: true
+  type: boolean
+  title: Keep or remove non-polar hydrogen atoms
+  short: If set to true, non-polar hydrogen atoms will be discarded to save computing
+    time
+  long: Since HADDOCK uses a united atom force field, the non-polar hydrogen atoms
+    can be in principle discarded. This saves computing time. However this should
+    not be done if you are defining distance restraints to specific hydrogen atoms
+    (e.g. when using NMR distance restraints).
+  group: molecule
+  explevel: easy
+log_level:
+  default: quiet
+  type: string
+  minchars: 0
+  maxchars: 100
+  choices:
+    - verbose
+    - normal
+    - quiet
+  title: Log level verbosity for CNS
+  short: Set the log level verbosity for CNS
+  long: CNS, the computational engine used by HADDOCK can generate a lot of output
+    messages. This parameter controls the verbosity of CNS (verbose, normal or quiet).
+  group: module
+  explevel: guru
+iniseed:
+  default: 917
+  type: integer
+  min: 0
+  max: 9999999999999999
+  title: Random seed
+  short: Random seed used in CNS to initialize the random seed function
+  long: Random seed used in CNS to initialize the random seed function
+  group: molecule
+  explevel: guru
+ligand_param_fname:
+  default: ''
+  type: file
+  title: Custom ligand parameter file
+  short: Ligand parameter file in CNS format
+  long: Ligand parameter file in CNS format containing all force field parameters
+    (bond, angles, dihedrals, impropers, van der waals) for any ligand not supported by default by HADDOCK
+  group: force field
+  explevel: expert
+ligand_top_fname:
+  default: ''
+  type: file
+  title: Custom ligand topology file
+  short: Ligand topology file in CNS format
+  long: Ligand topology file in CNS format containing the ligand topologies
+    (atoms, masses, charges, bond definitions...) for any ligand not supported by default by HADDOCK
+  group: force field
+  explevel: expert
```

Comment:
Can Martini handle small molecules ?
Can it be hybrid AllAtom parameters + Martini ?

Reply:
Not all or any small molecules, I think there is a set amount, at least for Martini2 (Martini3 has the possibility to add new small molecules, but that is not something that we can implement I think).
For hybrid, I think with CNS we could do it, but the last time we talked about it with @amjjbonvin it wasn't a priority.